### PR TITLE
Add post content

### PR DESCRIPTION
**Pull request by cansirin**
This PR introduces "content" field for Post model. From now on, content and site is optional fields for Post but Post is required to have at least URL or content.

Fixes #153 Implements #140

Post:
image

Send post:
image

Edit post:
image

### CODE REVIEW

**Review by rasitxy — APPROVED** [review]
LGTM

File: apps/pano/src/utils/index.ts [hunk 1]
```diff
@@ -79,11 +79,16 @@ export function validateURL(url: string) {
     return true;
   }
 }
+
+export function validate(value: unknown): value is string {
```
Comment (usirin) [inline apps/pano/src/utils/index.ts:83]: let's update this to accept the length as an optional argument (default: 0) and compare the trimmed length with that argument.

Comment (usirin) [reply apps/pano/src/utils/index.ts:83]: Let's do this with a fast-follow PR.

**Review by usirin — APPROVED** [review]

**Review by usirin — APPROVED** [review]
Lets goooo 💯 one more step before stack overflow like experience.

**Pull request merged**